### PR TITLE
Tutorial: How To Install CloudPanel on Debian 10

### PR DESCRIPTION
**Pull request by azadshaikh**
CloudPanel is a less-known web hosting control panel. CloudPanel V2 is coming soon. Till then this tutorial is best to get started with CloudPanel.

I have read and understood the Contributor's Certificate of Origin available at the end of
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.

Signed-off-by: Azad Shaikh [mohd.azad.shaikh@gmail.com](mailto:mohd.azad.shaikh@gmail.com)

### CODE REVIEW

File: tutorials/how-to-install-cloudpanel-debain-10/01.en.md [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+```
+
+You can see as below output :
+
+![Hostname](images/hostname.png)
+
+**Now, we will update our server using the below command.**
+
+```shell
+yum update -y
```
[the remark this answers is not in the document]
Comment (azadshaikh) [reply tutorials/how-to-install-cloudpanel-debain-10/01.en.md:83]: sudo apt update && sudo apt upgrade

 - sorry :(

File: tutorials/how-to-install-cloudpanel-debain-10/01.en.md [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+```
+
+**Now install Wget on the server.**
```
[the remark this answers is not in the document]
Comment (azadshaikh) [reply tutorials/how-to-install-cloudpanel-debain-10/01.en.md:86]: sudo apt update && sudo apt upgrade

 - sorry :(

File: tutorials/how-to-install-cloudpanel-debain-10/01.en.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (rook59) [reply tutorials/how-to-install-cloudpanel-debain-10/01.en.md:83]: No worries, can you just correct the two lines please?

**Comment by azadshaikh** [comment]
Fixed! Sorry for the delay.

**Review by rook59 — APPROVED** [review]

**Pull request merged**